tables/rubin_early_dr_scenario.tex

```diff
@@ -3,44 +3,47 @@
+Raw Images   &   \mycirc[RubinDarkTeal]  &   \mycirc[RubinDarkTeal]  &    --  &   \mycirc[RubinDarkTeal]  & \mycirc[RubinDarkTeal] &    \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  \arrayrulecolor{gray}\hline
+Processed Visit Images and Catalogs         &    \mycirc[RubinDarkTeal]  &   \mycirc[RubinDarkTeal]  &    --  &   \mycirc[RubinDarkTeal]  & \mycirc[RubinDarkTeal] &   \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]    &    \mycirc[RubinDarkTeal]  \\\hline 
+Coadded Images      &   \mycirc[RubinDarkTeal]  &   \mycirc[RubinDarkTeal]  &    --  &   --  & -- &    \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\\hline
+Object and ForcedSource Catalogs      &   \mycirc[RubinDarkTeal]  &   \mycirc[RubinDarkTeal]  &    --  &   --  &  -- &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &   \mycirc[RubinDarkTeal]  \\ \hline
+Difference Images and DIASources       &  --  &   \mycirc[RubinDarkTeal]  &    --  &   --  & \mycirc[RubinDarkTeal] &   \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  \hline
+ForcedSource Catalogs including DIA &  --  &   \mycirc[RubinDarkTeal]  &    --  &   --  & -- &   \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  \hline
 %PP Processed Visit Images     & --  &   --  &    --  &   --  &    --  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]   &    \mycirc[RubinDarkTeal]   \\ \hline 
 %PP Difference Images     & --  &   --  &    --  &   --  &    --  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]   &    \mycirc[RubinDarkTeal]  \\  \hline
 %PP Catalogs     &   --  &  -- &    --  &   --  &   \mycirc[RubinDarkTeal] &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]    &    \mycirc[RubinDarkTeal] \\  \hline
 % PP SSP Catalogs   &   --  &   -- &    \mycirc[RubinDarkTeal]   &   --  &    \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]   &    \mycirc[RubinDarkTeal]  \\  \hline
-DRP SSP Catalogs   &   --  &   -- &    --  &   --  &   --  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  
+Alerts &   --  &   -- &  --  &   --  & \mycirc[RubinDarkTeal] &  --  &  --  &  -- & --   \\ \hline 
+SSP Catalogs   &   --  &   -- &    \mycirc[RubinDarkTeal]  &   \mycirc[RubinDarkTeal]  & \mycirc[RubinDarkTeal] &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  
  \arrayrulecolor{black}\hline
 \end{tabular}}
-\caption{Summary of Data Release data products expected in each data preview and early LSST data release.}
+\caption{Summary of data products expected in each early LSST data release.}
 \label{tab:data-preview-summary}
 \end{table}
```

Comment:
The rework of this table was intended to include only DRP data products, with another section to be added for PP data products. A Data Preview is made with DRP  ... AP and PP are additional components of the ES program.

Reply:
I understand the rationale for only including DRPs in the table--I just worry that it leads people on- and off-project to forget where AP comes in the sequence.  The new timeline figure later in the document helps some if we are sure to show it.

Reply:
Good point, let's merge this branch as you propose and we can then take a more global look at how we organise and present the data products